maintainers/maintainer-list.nix

```diff
@@ -10692,6 +10692,15 @@
     githubId = 8214542;
     name = "Nicolò Balzarotti";
   };
+  nicoo = {
+    email = "nicoo@debian.org";
+    gihub = "nbraud";
```

Comment:
```suggestion
    github = "nbraud";
```

Reply:
Thanks for catching that, was still trying to interpret ofborg's output  :sweat_smile: